### PR TITLE
Streams 1750 - document new REST api paths

### PR DESCRIPTION
**Pull request by cremond**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

In Streams 1750 api paths were changed to anticipate a need to version them. This PR is meant to include those changes in the documentation.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

**Comment by cremond** [comment]
@jfmarquet ready for your review

**Comment by jfmarquet** [comment]
Review ok for me.

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -516,11 +515,11 @@ my-release-subscriber-webhook-84469bd68f-lqxgk                 1/1     Running
 In order to check that Streams is running:
 
 1. Create a topic with default settings using the provided Postman collection and Postman environment.
-As the provided environment is configured with localhost, you may need to reconfigure it with your own DNS, for instance `baseUrl` will be changed from `http://localhost:9001` to `https://k8s.yourdomain.tld` and so on for other variables.
+As the provided environment is configured with `${loadbalancer.baseUrl}` for all base URLs, you need to reconfigure it with your own DNS, for instance `baseUrl` will be changed from `${loadbalancer.baseUrl}` to `https://k8s.yourdomain.tld` and so on for other variables.
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:518]:
```suggestion
As the provided environment is configured with `${loadbalancer.baseUrl}` for all base URLs, you must reconfigure it with your own DNS. For example, `baseUrl` will be changed from `${loadbalancer.baseUrl}` to `https://k8s.yourdomain.tld` and so on for other variables.
```

File: content/en/docs/publishers/publisher-http-poller.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ The http-poller publisher requires some specific configuration.
 
 ### Removing http headers from configuration
 
-In order to remove a header from publisher's configuration, set its value to `null` when calling `PATCH /topics/{{topicId}}` endpoint:
+In order to remove a header from publisher's configuration, set its value to `null` when calling `PATCH /streams/hub/api/v1/topics/{{topicId}}` endpoint:
```
Comment (lbadenhop) [inline content/en/docs/publishers/publisher-http-poller.md:56]:
```suggestion
To remove a header from publisher's configuration, set its value to `null` when calling `PATCH /streams/hub/api/v1/topics/{{topicId}}` endpoint:
```

File: content/en/docs/publishers/publisher-http-post.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Streams will then take care of storing and forwarding events to the subscribers.
 
 ### http-post publisher configuration
 
-In order to create a Streams topic associated to a http-post publisher, you must perform a request on `POST /topics` endpoint.
+In order to create a Streams topic associated to a http-post publisher, you must perform a request on `POST /streams/hub/api/v1/topics` endpoint.
```
Comment (lbadenhop) [inline content/en/docs/publishers/publisher-http-post.md:16]:
```suggestion
To create a Streams topic associated to a http-post publisher, you must perform a request on `POST /streams/hub/api/v1/topics` endpoint.
```

File: content/en/docs/publishers/publisher-http-post.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ The http-post publisher does not require any specific configuration. The config
 
 Once the topic is created, the external component can publish payloads using the following endpoint:
 
-`POST /publishers/http-post/topics/{{topicId}}`
+`POST /streams/publishers/http-post/api/v1/topics/{{topicId}}`
 
 Note that only `application/json` content is accepted. Below an example of a valid json payload:
```
Comment (lbadenhop) [inline content/en/docs/publishers/publisher-http-post.md:36]:
```suggestion
Note that only `application/json` content is accepted. Below is an example of a valid json payload:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ Below the list of HTTP status codes that can be returned when deleting the kafka
 
 In order to get an existing subscription, simply do the following GET request:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:68]:
```suggestion
To get an existing subscription, use the following GET request:
```

File: content/en/docs/subscribers/subscriber-kafka.md [hunk 6]
```diff
@@ -100,7 +100,7 @@ The following HTTP status codes can be returned while testing a Kafka subscripti
 
 Use the following `GET` request on your topic to get existing subscriptions:
 
-`GET /subscribers/kafka/topics/{topicId}/subscriptions`
+`GET /streams/subscribers/kafka/api/v1/topics/{topicId}/subscriptions`
 
 See [pagination](/docs/topics-api/#pagination) to get more information about how pagination and sorting work.
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-kafka.md:105]:
```suggestion
See [pagination](/docs/topics-api/#pagination) for more information on how pagination and sorting work.
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 7]
```diff
@@ -50,7 +50,7 @@ Below the list of HTTP status codes that can be returned when trying to create a
 
 In order to stop the sending of webhook notifications, simply delete the corresponding webhook subscription with following request:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:51]:
```suggestion
To stop the sending of webhook notifications, simply delete the corresponding webhook subscription with following request:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 8]
```diff
@@ -65,7 +65,7 @@ Below the list of HTTP status codes that can be returned when deleting the webho
 
 In order to get existing subscription, simply do the following GET request:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:66]:
```suggestion
To get an existing subscription, use the following GET request:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 9]
```diff
@@ -160,7 +160,7 @@ The following HTTP status codes can be returned when interacting with the exchan
 
 To get existing subscriptions, do the following GET request on your topic:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:161]:
```suggestion
To get existing subscriptions, use the following GET request on your topic:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 10]
```diff
@@ -160,7 +160,7 @@ The following HTTP status codes can be returned when interacting with the exchan
...
 
-`GET /subscribers/webhook/topics/{topicId}/subscriptions`
+`GET /streams/subscribers/webhook/api/v1/topics/{topicId}/subscriptions`
 
 See [pagination](/docs/topics-api/#pagination) to get more information about how pagination and sorting works.
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:165]:
```suggestion
See [pagination](/docs/topics-api/#pagination) for more information on how pagination and sorting works.
```

File: content/en/docs/topics-api.md [hunk 11]
```diff
@@ -44,11 +25,11 @@ You can specify how many items to receive (up to a maximum of `1000`) via the `p
 All paginated queries start at page `1`.
 Pagination information is provided by the `Link` header of a response.
 
-For example, let's make a request to the `GET /api/v1/topics` endpoint with the `pageSize` query param set to `5`.
-The `Link` header will contain a list of element separated by comma pointing to the different pages allowing you to navigate easily.
+For example, let's make a request to the `GET /streams/hub/api/v1/topics` endpoint with the `pageSize` query param set to `5`.
+The `links` section of the response body will contain a list of element separated by comma pointing to the different pages allowing you to navigate easily.
```
Comment (lbadenhop) [inline content/en/docs/topics-api.md:29]:
```suggestion
The `links` section of the response body will contain a list of elements, separated by commas, pointing to the different pages, allowing you to navigate easily.
```

**Review by lbadenhop — APPROVED** [review]

**Comment by cremond** [comment]
Thanks @lbadenhop for your swift review ! I have applied all of your suggestions in the latest commit.

**Review by lbadenhop — APPROVED** [review]

**Review by allandenis38 — APPROVED** [review]

**Pull request merged**